### ISSUE TITLE
raw create game form

### ISSUE THREAD

**Issue by joram** [battlesnakeio/roadmap#58]
this is done via a post to the engine endpoint.

**Comment by CDFriend** [comment]
I'll try taking this on

**Comment by CDFriend** [comment]
So I'm just looking at submitting the form data to the server, and was wondering a couple things:
1. Should the POST request to the engine be sent from the client side?
2. What should be put in the `SnakeOptions.Id` field of the request? Won't the ID of the snake be generated by the engine?

**Comment by dlsteuer** [comment]
I would post the form back to the django backend, and have the backend create the form, this way you don't have to deal with browser side cors issues for now. For the second item, the engine will generate an id if it's not provided, but we want the play side to be the source of truth for snake id's. So when a snake is created for a user/team, it would be assigned an ID on the django server.

**Comment by CDFriend** [comment]
Perfect, thanks. Just to clarify, this is like the form we used to create games in the engine last year?

Screenshot of what I have right now:  
<img width="1158" alt="screen shot 2018-04-27 at 4 00 41 pm" src="https://user-images.githubusercontent.com/8661121/39398723-a37b719c-4ac6-11e8-9d7c-40195d79931f.png">

**Comment by dlsteuer** [comment]
Yup that's correct and looks good.

**Comment by joram** [comment]
@CDFriend how's that branch going?

**Comment by CDFriend** [comment]
Sorry, got caught up with other things here...I currently have the HTML form and a way of sending a request to the engine directly from the browser. 

Might still be a couple weeks before I can get this done - if you'd like I can open a PR with what I have?

**Comment by codeallthethingz** [comment]
@CDFriend yes please - connect this issue to the PR and we'll take a look.

**Comment by CDFriend** [comment]
@codeallthethingz Created a pull request here: https://github.com/battlesnakeio/play/pull/8

**Comment by CDFriend** [comment]
I'll keep working on the POST to the engine in this ticket. There'll probably be some refactoring of the HTML here, as I'll have to integrate with the `django.forms` module to send form data back to the server.

**Comment by CDFriend** [comment]
@codeallthethingz Looks like this may have been covered in https://github.com/battlesnakeio/play/pull/10 - should we close this ticket?
